content/en/agent/guide/private-link.md

```diff
@@ -231,6 +231,245 @@ The VPCs with Private Hosted Zone (PHZ) attached need to have a couple of settin
+| API                       | `com.amazonaws.vpce.ap-northeast-1.vpce-svc-008cd79a7a09e0a1e` | `api.ap1.datadoghq.com`                    |
+| Process                   | `com.amazonaws.vpce.ap-northeast-1.vpce-svc-0e86b29a0dc2c8a26` | `process.ap1.datadoghq.com`                |
+| Profiling                 | `com.amazonaws.vpce.ap-northeast-1.vpce-svc-0d598057ecde10596` | `intake.profile.ap1.datadoghq.com`         |
+| Traces                    | `com.amazonaws.vpce.ap-northeast-1.vpce-svc-0a5d94b2e8f6e70fc` | `trace.agent.ap1.datadoghq.com`            |
+| Containers                | `com.amazonaws.vpce.ap-northeast-1.vpce-svc-03ffd8d552f0d957d` | `orchestrator.ap1.datadoghq.com`           |
+
+You can also find this information by interrogating the AWS API, `DescribeVpcEndpointServices`, or by using the following CLI command: `aws ec2 describe-vpc-endpoint-services --service-names <service-name>`.
+
+For example, in the case of the Datadog metrics endpoint:
+
+  ```bash
+  aws ec2 describe-vpc-endpoint-services --service-names com.amazonaws.vpce.ap-northeast-1.vpce-svc-002d904d5e69340ad | jq '.ServiceDetails[0].PrivateDnsName'
+  ```
+
+This returns `metrics.agent.ap1.datadoghq.com`, the private hosted zone name that you need in order to associate with the VPC which the Agent traffic originates in. Overriding this record grabs all Metrics-related intake hostnames.
+
+2. Within each new Route53 private hosted zone, create an A record with the same name. Toggle the **Alias** option, then under **Route traffic to**, choose **Alias to VPC endpoint**, **ap-northeast-1**, and enter the DNS name of the VPC endpoint associated with the DNS name.
+
+   **Notes**:
+  - To retrieve your DNS name, see the [View endpoint service private DNS name configuration documentation.][4]
+  - The Agent sends telemetry to versioned endpoints, for example, `<version>-app.agent.ap1.datadoghq.com` which resolves to `metrics.agent.ap1.datadoghq.com` through a CNAME alias. Therefore, you only need to set up a private hosted zone for `metrics.agent.ap1.datadoghq.com`.
+
+{{< img src="agent/guide/private_link/create-an-a-record.png" alt="Create an A record" style="width:90%;" >}}
+
+3. Configure VPC peering and routing between the VPC in `ap-northeast-1` that contains the Datadog PrivateLink endpoints and the VPC in the region where the Datadog Agents run.
+
+4. If the VPCs are in different AWS accounts, the VPC containing the Datadog Agent must be authorized to associate with the Route53 private hosted zones before continuing. Create a [VPC association authorization][5] for each Route53 private hosted zone using the region and VPC ID of the VPC where the Datadog Agent runs. This option is not available in the AWS Console. It must be configured using the AWS CLI, SDK, or API.
+
+5. Edit the Route53 hosted zone to add the non-us-east-1 VPC.
+
+{{< img src="agent/guide/private_link/edit-route53-hosted-zone.png" alt="Edit a Route53 private hosted zone" style="width:80%;" >}}
```

Comment:
Similar to the previous comment, we could update this screenshot to show AP1, but it's also not a big deal (and might be a headache to to change this for future supported regions anyways).